src/en/status.md

```diff
@@ -102,7 +102,7 @@ GET /v2/notifications
 
 #### All messages
 
-This will return all your messages with statuses. They will display in pages of up to 250 messages each.
+This will return your messages with statuses. The type of API key you use to make requests to this endpoint is important (i.e. Team, Test, or Live). This type will be matched with the type of API key that created your messages. If you make a request with a Live API key, only messages created by Live API keys will be returned. They will display in pages of up to 250 messages each.
```

Comment:
@smcmurtry so are you saying the way it works is if you hit the `/notifications` endpoint with a live API key you get all notifications that were sent with ANY live API key, not just the one you authenticated with? 🤔

Reply:
Yes that is correct! Here is the logic: https://github.com/cds-snc/notification-api/blob/main/app/dao/notifications_dao.py#L262

Reply:
If we do it that way, it would break the use case of "I just want to know the status of all the notifications for my service, so I can be alerted if any have failed". I'll send you a link to the support ticket where this seemed to be their use case.

Reply:
Interesting -- I also assumed it worked the way that @andrewleith thought. While it is not a major breach (same service with same permissions), that looks counter-intuitive to me. Practically, most services have one live API key anyway so that wouldn't break much. 

In a user scenario with several roles and different permissions scheme, the current way of working could be an issue. What if a service hires a non-gov third party and creates a live key especially for them to send a newsletter, but then they'd have access to all notifications of that service? This would break privacy policies. The only way with Notify to avoid this at the moment is to create an entirely different service then: the API key itself isn't enough to protect notifications sent with another one that might be more sensitives.

Not a blocker for this PR as we are simply explaining current way it works -- so I'll approve.